### ISSUE TITLE
[iOS] Xsolla SDK should respect existing plist values

### ISSUE THREAD

**Issue by artyom-v** [xsolla/store-unity-sdk#70]
Xsolla SDK v1.5.0

Xsolla iOSBuildPostprocess.cs overwrites the existing values in the list. It breaks other functions.
```
		private static void SetupDeepLinking(string outputPath)
		{
			var plistPath = Path.Combine(outputPath, "Info.plist");
			var plistDoc = new PlistDocument();
			plistDoc.ReadFromString(File.ReadAllText(plistPath));

			var rootDic = plistDoc.root;
			var urlTypesArray = rootDic.CreateArray("CFBundleURLTypes");

			var urlNamesDic = urlTypesArray.AddDict();
			urlNamesDic.SetString("CFBundleURLName", "");

			var urlSchemesArray = urlNamesDic.CreateArray("CFBundleURLSchemes");
			urlSchemesArray.AddString("app");

			File.WriteAllText(plistPath, plistDoc.WriteToString());
		}
```
This code completely destroy CFBundleURLTypes array and create new. It should add values to already existing array. Same for CFBundleURLSchemes array.

Please fix.

**Comment by DenisRakhimovXsolla** [comment]
Hey @artyom-v 
Thanks for the report!
This fix will be included in the next release, which is scheduled for next week.

**Comment by DenisRakhimovXsolla** [comment]
Fixed in version 2.1.0
